Profile_Properties_Definition.md

```diff
@@ -0,0 +1,234 @@
+# Profile Properties Definition
+
+**Status**: WIP
+
+**Version**: Alpha
+
+**Author(s)**: Leon Wang (@leonwanghui)
+
+## Summary
+
+This proposal is designed for standardizing the properties definition of OpenSDS
+profile. This design is also part of referenced implementation of LineOfService
+defined in [Swordfish](https://www.snia.org/sites/default/files/SMI/swordfish/v106/Swordfish_v1.0.6_Specification.pdf)
+specification (v1.0.6), which promises to provide a unified approach for the
+management of storage and servers in hyperscale and cloud infrastructure
+environments.
```

Comment:
Please reference the google doc on profiles as well.

Reply:
Hi, I pasted the link in the PR description section, but I can update the link in the proposal.